_includes/prod_deployment/synchronize-clocks.md

```diff
@@ -0,0 +1,145 @@
+CockroachDB requires moderate levels of [clock synchronization](recommended-production-settings.html#clock-synchronization) to preserve data consistency. For this reason, when a node detects that its clock is out of synch with at least half of the other nodes in the cluster by 80% of the maximum offset allowed (500ms by default), it spontaneously shuts down. This avoids the risk of consistency anomalies, but it's best to prevent clocks from drifting too far in the first place by running clock synchronization software on each node.
+
+{% if page.title contains "Digital Ocean" or page.title contains "Manual" %}
+
+[NTP](http://www.ntp.org/) should keep offsets in the single-digit milliseconds, so that software is featured here, but other methods of clock synchronization are suitable as well.
+
+1. SSH to the first machine.
+
+2. Disable `timesyncd`, which tends to be active by default on some Linux distributions:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ sudo timedatectl set-ntp no
+    ~~~
+
+    Verify that `timesyncd` is off:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ timedatectl
+    ~~~
+
+    Look for `Network time on: no` or `NTP enabled: no` in the output.
+
+3. Install NTP:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ sudo apt-get install ntp
+    ~~~
+
+4. Force the machine's clock to synch with Google's NTP service:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ sudo service ntp stop
+    ~~~
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ sudo ntpd -b time.google.com
```

Comment:
`ntpdate`, not `ntpd`. We should also mention why we're doing that.
`ntpdate` also need to be installed, it's not included in the `ntp` package. So the apt-get line becomes: `sudo apt-get install ntpdate ntp`.

If we're using `time.google.com`, then we need to modify `/etc/ntp.conf` as well.

Reply:
Making this change permanent involves changing `/etc/ntp.conf` and restarting the service.
Google [has docs](https://developers.google.com/time/guides#linux_ntpd) on this for various systems.
For ubuntu, the new file looks like:
```
# Specify one or more NTP servers.

# Use servers from the NTP Pool Project. Approved by Ubuntu Technical Board
# on 2011-02-08 (LP: #104525). See http://www.pool.ntp.org/join.html for
# more information.
#pool 0.ubuntu.pool.ntp.org iburst
#pool 1.ubuntu.pool.ntp.org iburst
#pool 2.ubuntu.pool.ntp.org iburst
#pool 3.ubuntu.pool.ntp.org iburst

# Use Ubuntu's ntp server as a fallback.
#pool ntp.ubuntu.com

server time1.google.com iburst
server time2.google.com iburst
server time3.google.com iburst
server time4.google.com iburst
```

Where I removed the `ntp.ubuntu.com` and switched to using the servers specified in the Google docs.